src/main/java/io/debezium/connector/yugabytedb/YugabyteDBConnector.java

```diff
@@ -71,7 +73,7 @@ public Class<? extends Task> taskClass() {
     @Override
     public void start(Map<String, String> props) {
         this.props = props;
-        LOGGER.debug("Props " + props);
+        LOGGER.info("Props " + props);
```

Comment:
I think we can change this back to debug, the properties are anyway printed once. With this we might end up printing all the properties twice. Please verify once and change back if that's the case.